backend/src/tasks/helpers/parseCommandOptions.ts

```diff
@@ -0,0 +1,18 @@
+import { CommandOptions } from '../ecs-client';
```

Comment:
You really shouldn't be putting this as a default export in a generic library named parseCommandOptions. Is this how you parse all command options? If we were going to do it this way, what if we wanted to parse the command options of another function with different arguments?

Reply:
I'm not sure what you want me to do for item 1, but I've refactored parseCommandOptions to sanitizeChunkValues, as requested.

Reply:
I've gone back to the helper directory, and see that this is consistent with existing functionality. I'm okay with this.